isis/src/lro/apps/lrowac2isis/lrowac2isis.cpp

```diff
@@ -0,0 +1,846 @@
+/** This is free and unencumbered software released into the public domain.
+
+The authors of ISIS do not claim copyright on the contents of this file.
+For more details about the LICENSE terms and the AUTHORS, you will
+find files of those names at the top level of this repository. **/
+
+/* SPDX-License-Identifier: CC0-1.0 */
+
+#include "lrowac2isis.h"
+
+#include <cstdio>
+#include <QString>
+
+#include <QRegExp>
+
+#include "ProcessImportPds.h"
+#include "LineManager.h"
+#include "OriginalLabel.h"
+#include "Brick.h"
+#include "History.h"
+#include "Stretch.h"
+#include "Application.h"
+
+using namespace std;
+
+namespace Isis {
+
+  vector<Cube *> outputCubes;
+  vector<int> frameletLines;
+  void separateFramelets(Buffer &in);
+  void writeNullsToFile();
+  void TranslateLabels(Pvl &pdsLab, Pvl &isis3VisEven, Pvl &isis3VisOdd,
+                       Pvl &isis3UvEven, Pvl &isis3UvOdd, UserInterface &ui);
+  void ValidateInputLabels(Pvl &pdsLab);
+
+  std::vector<int> padding;
+  int colorOffset = 0;
+  int inputCubeLines = 0;
+
+  // Output UV Files
+  Cube *uveven = NULL;
+  Cube *uvodd = NULL;
+
+  // Output VIS Files
+  Cube *viseven = NULL;
+  Cube *visodd = NULL;
```

Comment:
These are pretty sus and would be the first thing I would look at.

Reply:
updated Cubes to static.